vignettes/geological-completeness.Rmd

```diff
@@ -104,49 +104,49 @@ marginal <- subset(environments, class == "marginal")
+Great! So we now have our groupings, but if we want to investigate how lithostratigraphic units change through time, we need a little more information. Specifically, we need to know the inferred paleoenvironment and age of each lithostratigraphic unit. We can retrieve such information about lithostratigraphic units using the `get_units` function. Conveniently, we can request such data for each paleoenvironmental class using the `environ_id` column.
 
 *Hint: Remember that to retrieve data stored in [Macrostrat](https://macrostrat.org) (e.g., Macrostrat columns, sections, and units), the `get_` suite of functions must be used.*
 
 ```{r get_units, eval = TRUE}
 # Retrieve marine units
-marine <- get_units(environ_id = marine$environ_id)
+marine_units <- get_units(environ_id = marine$environ_id)
 # Retrieve marginal units
-marginal <- get_units(environ_id = marginal$environ_id)
+marginal_units <- get_units(environ_id = marginal$environ_id)
 # Retrieve terrestrial units
-terrestrial <- get_units(environ_id = terrestrial$environ_id)
+terrestrial_units <- get_units(environ_id = terrestrial$environ_id)
 # Let's also add the environment class in these data.frames for consistency
-marine$environ_class <- "Marine"
-marginal$environ_class <- "Marginal"
-terrestrial$environ_class <- "Terrestrial"
+marine_units$environ_class <- "Marine"
+marginal_units$environ_class <- "Marginal"
+terrestrial_units$environ_class <- "Terrestrial"
 ```
 
 Nice! That was pretty straight forward. However, we notice something weird. Some `unit_id` values in the `marine` dataframe are also present in the marginal and terrestrial dataframes, and vice versa:
 
 ```{r duplicate_ids, eval = TRUE}
 # How many marine unit IDs are also in terrestrial?
-length(which(marine$unit_id %in% terrestrial$unit_id))
+length(which(marine_units$unit_id %in% terrestrial_units$unit_id))
 ```
 
-On investigation, we notice that some lithostratigraphic units are composed of multiple paleoenvironments (various combinations of marine, marginal, and terrestrial). Interesting, but to be expected given transgression and regression cycles! We should treat these differently to our pre-existing groupings as they clump together different paleoenvironments which could capture broader information (e.g., terrestrial and marine organisms). Let's make a new paleoenvironment class called "mixed", and update the environment class accordingly.
+On investigation, we notice that some lithostratigraphic units are associated with multiple different paleoenvironments (various combinations of marine, marginal, and terrestrial). Interesting, but perhaps to be expected given transgression and regression cycles! We should treat these differently to our pre-existing groupings for clarity. Let's make a new paleoenvironment class called "mixed", and update the environment class accordingly.
```

Comment:
Peters and Heim also include units with "unknown" paleoenvironments. Should we do that here?

Reply:
Hmmm, I think this would require setting up this vignette in a slightly different way... downloading all units, and then splitting them based on their palaeoenvironment. I get the want for wanting to include "unknown" palaeoenvironments, but I think we're currently showing off how to interact with `rmacrostrat` through querying in a way which will be more useful for users. I could also download all units, and filter units based on the unit IDs we have for the other palaeoenvironments.

Reply:
I'll leave it up to you. I just wanted to note that the vignette is started by saying we are going to reproduce the figure, then we make a slightly different figure.